assets/js/weatherSript.js

```diff
@@ -0,0 +1,85 @@
+console.log("Hello World!!!")
+
+var apiKey = '13765804293c80fb9eac8b6b1d0beeb5';
+var locationName; // will store user input based on location
+var locationForm = document.querySelector('#location-form');
+var locationInput = document.querySelector('#location-input');
+
+var submitLocationInput = function(event) {
+    event.preventDefault();
+    locationName = locationInput.value.trim();
+    console.log('user entered: ' + locationName);
+    firstCall(locationName);
+};
+
+var firstCall = function(location) {
+    
+    // check if the input isn't empty string and make a call
+    if (locationName) {
+        var firstURL = 'https://api.openweathermap.org/data/2.5/weather?q='+locationName+'&appid='+apiKey+'&units=imperial';
+        fetch(firstURL).then(function(response) {
+            // check if entered location name actually exists
+            if(response.ok) {
+                response.json().then(function(data) {
+                    // call getGeoLocation to get longitude and latitude for the precise weather at a requested location
+                    console.log(data);
+                    getGeoLocation(locationName, data);
+                });
+            } else {
+                alert('sorry, location with this name not found');
+            }
+        })
+        // show an alert in case there's network error
+        .catch(function(error) {
+            alert('unable to connect to server. try again later');
+        })
+        // empty the input field
+        locationInput.value = '';
+
+    // show an alert if user submits empty field    
+    } else {
+        alert('Please enter a location name');
+    }
+    
+};
+
```

Comment:
^ might want to fix some of the code organization on this if else statement here to improve readability.